internal/policy/commit/commit.go

```diff
@@ -33,6 +33,10 @@ type Commit struct {
 	RequireCommitBody bool `mapstructure:"requireCommitBody"`
 	// Conventional is the user specified settings for conventional commits.
 	Conventional *Conventional `mapstructure:"conventional"`
+	// HeaderCase is the case that the first word of the header must have ("upper" or "lower").
+	HeaderCase string `mapstructure:"headerCase"`
+	// HeaderInvalidLastCharacters is a string containing all invalid last characters for the header.
+	HeaderInvalidLastCharacters string `mapstructure:"headerInvalidLastCharacters"`
```

Comment:
Should we have a set of default characters?

Reply:
Hmm, how would that show up in config? I played around with the `mapstructure` definitions and I didn't see a good way to expose a default.